main.py

```diff
@@ -15,7 +16,8 @@ def main():
         if locked:
             try:
                 # Test connection to BTC (by extension LND)
-                locked = umbrui.btc_grpc.connection_locked()
+                btc_rpc = BtcRPC()
```

Comment:
This is good! Because we don't need to instantiate the UI until it is ready to go

Reply:
I actually just did this because I had to move the connection outside the UI to make the code cleaner